package.json

```diff
@@ -28,20 +30,26 @@
     "eventemitter3": "2.0.3",
     "hash-brown-router": "~3.2.0",
     "iso-next-tick": "1.0.0",
-    "native-promise-only": "0.8.1",
     "page-path-builder": "~1.0.3",
     "path-to-regexp-with-reversible-keys": "~1.0.3",
-    "then-denodeify": "1.0.1",
-    "xtend": "^4.0.0"
+    "then-denodeify": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.26.0",
+    "babel-plugin-external-helpers": "6.22.0",
+    "babel-preset-es2015": "6.24.1",
     "browserify": "14.0.0",
     "covert": "^1.1.0",
     "faucet": "0.0.1",
+    "rollup": "0.50.0",
+    "rollup-plugin-babel": "3.0.2",
+    "rollup-plugin-commonjs": "8.2.1",
+    "rollup-plugin-node-resolve": "3.0.0",
+    "rollup-plugin-visualizer": "0.3.1",
+    "rollup-watch": "4.3.1",
     "tap-browser-color": "0.1.2",
     "tape": "^4.1.0",
     "tape-catch": "1.0.6",
-    "watchify": "3.9.0",
-    "webpack": "^1.12.14"
```

Comment:
It was an old version of Webpack, so it wasn't really testing anything relevant any more.

The Webpack test was added after `require`ing a JSON file caused Webpack to break at some point a while back.  (It would cause default Rollup setups to break, too).

Since I'm bundling everything except for some external dependencies now, there's not much risk of breaking bundlers any more.